frontend/packages/noobaa-storage-plugin/src/components/object-data-reduction-card/object-data-reduction-card-item.tsx

```diff
@@ -35,7 +34,15 @@ export const EfficiencyItem: React.FC<EfficiencyItemProps> = React.memo(
   ({ efficiency, isLoading, error }) => {
     const infoText =
       'Efficiency ratio refers to the deduplication and compression process effectiveness.';
-    const stats: string = !_.isNil(efficiency) ? `${Number(efficiency).toFixed(1)}:1` : null;
+    let stats: string = efficiency;
+    let efficiencyValue = Number(efficiency);
+    // To one decimal place
+    if (!Number.isInteger(Number(efficiency))) {
+      efficiencyValue = Number(efficiencyValue.toFixed(1));
+    }
+    if (efficiency) {
+      stats = efficiencyValue === 0 ? '1:1' : `${efficiencyValue}:1`;
```

Comment:
```suggestion
    if (efficiency) {
      const efficiencyValue = +Number(efficiency).toFixed(1);
      stats = efficiencyValue === 0 ? '1:1' : `${efficiencyValue}:1`;
    }
```

Reply:
Can change `efficiencyValue` to `formattedEfficiency`

Reply:
`efficiencyValue` is shared between two blocks it needs to be declared on the outermost block.

Reply:
This is a replacement for
```
 let stats: string = efficiency;
    let efficiencyValue = Number(efficiency);
    // To one decimal place
    if (!Number.isInteger(Number(efficiency))) {
      efficiencyValue = Number(efficiencyValue.toFixed(1));
    }
    if (efficiency) {
      stats = efficiencyValue === 0 ? '1:1' : `${efficiencyValue}:1`;
```

Reply:
You do realize that your code would give me 2.0:1. And that is what you asked me to remove.

Reply:
How ?
Can you explain ?

Reply:
I didn't know `+` would remove the floating `.0`. Thank you for the suggestion.